server/controllers/file.controller.ts

```diff
@@ -56,18 +58,23 @@ export class FileController {
     if (fs.existsSync(filename)) {
       const loadedFile = fs.readFileSync(filename);
 
-      return loadedFile;
+      const contentType = this._getContentTypeFromExtension(filename);
+
+      //return loadedFile
+      response.header({
+        'Content-type': contentType
+      }).status(200).send(loadedFile);
```

Comment:
im surprised it doesn't break without a return  - since right after that send it throws an error :D 

could you also try to use return instead of the send method? ... it "should" work then too

otherwise NICE!